examples/java/gradle-shared-module/build.gradle

```diff
@@ -0,0 +1,44 @@
+import groovy.json.JsonSlurper
+
+subprojects {
+
+    apply plugin: 'java'
+
+    repositories {
+        mavenCentral()
+        maven {
+            name = "GitHubPackages"
+            url = uri("https://maven.pkg.github.com/hashicorp/terraform-cdk")
+            credentials {
+                username = project.githubUsername
+                password = project.githubToken
```

Comment:
@skorfmann I can update this line to pull the token from an environment variable and fallback to a project property if not found. We would need a token generated and added to travis CI. It would be preferable to only grant permissions for reading GitHub package repos.

Reply:
I'm pretty sure the github action runner already has a user/token with permissions, so adding to environment variables should be pretty easy.

Reply:
Sorry about that, I was thinking about another project when mentioning travis. Ill update the workflow now to add a reference to the token var